client/src/components/Forms/signup/index.jsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Modal,
+  Button,
+  Form,
+  Input,
+  message,
+} from 'antd';
+import logo from '../../../assets/images/logo.png';
+import './Signup.css';
+
+function Signup() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirm, setConfirm] = useState('');
+  const [rool] = useState('');
```

Comment:
Why do you need a role state here? in the sign-up form, it will always be `interviewee`